src/nvim/memline.c

```diff
@@ -1616,8 +1596,8 @@ static time_t swapfile_info(char_u *fname)
   return x;
 }
 
-/// Returns TRUE if the swap file looks OK and there are no changes, thus it
-/// can be safely deleted.
+/// @return  TRUE if the swap file looks OK and there are no changes, thus it
+///          can be safely deleted.
 static time_t swapfile_unchanged(char *fname)
```

Comment:
I just noticed this function had a wrong return type (<https://github.com/vim/vim/pull/10000>). That's may I marked this as draft again.

Reply:
Aha, thought it was a "you can undraft this" ping :). Redrafting again then, let me know when this is good to go.